src/cmds/qstat.c

```diff
@@ -1078,7 +1081,24 @@ display_statjob(struct batch_status *status, struct batch_status *prtheader, int
 						(void)decode_xml_arg_list_str((a->value),  &cmdargs);
 						prt_attr(a->name, a->resource, cmdargs);
 						printf("\n");
-					} else {
+					}
+					else if (strcmp(a->name, ATTR_executable) == 0) {
```

Comment:
Please follow the formatting style of the rest of the file and put the 'else if' after the preceding '}'